### ISSUE TITLE
downloadable?

### ISSUE THREAD

**Issue by luminosity-d3v** [yenbook/proxyearth#32]
I want to be able to download the proxy website, so I cannot open it as a file and it cannot be blocked; will you add further accessibility?
